### PR TITLE
[New model] Support MobileNetV3

### PR DESCRIPTION
**Pull request by yamengxi**

### CODE REVIEW

**Comment by yamengxi** [comment]
I built the network the same as https://github.com/ekzhang/fastseg, but get a lower result as following:
| config | mIoU(my result) | mIoU(fastseg) | mIoU(original paper) |
|-----------|----------|----------|----------|
| configs/mobilenet_v3/lraspp_m-v3-d8_512x1024_160k_cityscapes.py | 67.56 | 72.3 | 72.36 |
| configs/mobilenet_v3/lraspp_m-v3s-d8_512x1024_160k_cityscapes.py | 61.43 | 67.4 | 68.38 |

I don't know why my training results are lower than original paper (about 5~6 points).
Can anyone give me some advice and help?

**Comment by hellock** [comment]
Task linked: [CU-fjq4w0 MobileNetV3](https://app.clickup.com/t/fjq4w0)

File: .gitignore [hunk 1]
```diff
@@ -112,6 +112,8 @@ data
 *.pkl.json
 *.log.json
 work_dirs/
+old_work_dirs/
+configs/mobilenet_v3_test/
```
Comment (xvjiarui) [inline .gitignore:116]: These modification is not necessary.

File: mmseg/models/decode_heads/lraspp_head.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                1,
+                act_cfg=dict(type='Sigmoid'),
+                bias=False))
+
+    def forward(self, inputs):
+        """Forward function."""
+        inputs = self._transform_inputs(inputs)
+
+        x = inputs[-1]
+        assert x.size(2) >= 49 and x.size(3) >= 49
```
Comment (xvjiarui) [inline mmseg/models/decode_heads/lraspp_head.py:73]: Why is 49?

File: .gitignore (hunk 1 above)
Comment (yamengxi) [reply .gitignore:116]: OK, I will delete it in the final version.

File: mmseg/models/decode_heads/lraspp_head.py (hunk 2 above)
Comment (yamengxi) [reply mmseg/models/decode_heads/lraspp_head.py:73]: It is proposed in the paper.
image

Comment (yamengxi) [reply mmseg/models/decode_heads/lraspp_head.py:73]: We have to make sure that the input size is not less than 49.

Comment (xvjiarui) [reply mmseg/models/decode_heads/lraspp_head.py:73]: Will PyTorch raise the error if we don't assert? Or we may add comments here.

**Review by xvjiarui — APPROVED** [review]

**Pull request merged**